spec/soda_machine_spec.rb

```diff
@@ -40,7 +40,7 @@
         @sold_soda = soda_machine.sell('Coke Zero')
       end
       it "returns the sold soda" do
-        expect(@sold_soda).to be(coke_zero)
+        expect(@sold_soda).to eq(coke_zero)
```

Comment:
Come on! Don't change the tests! Big no no in development. Tests drive the code, not the other way around.